### DISCUSSION TITLE
MegaCode (SubGHz) Feature Request to generate brute force files.

### DISCUSSION THREAD

**Discussion by wp3gi** [FlipperMaker/flippermaker.github.io#27] [Ideas]
Hi... I'm interested in brute forcing MegaCode.  I wanted to request a feature that would make it easier since the BruteForce app doesn't have the option that I know of.

I would need to generate a set of files instead of just one .sub file.  Almost always the "facility" and the "button" is ignored by the receiver and what matters is just the code.  There could be two options to choose a value for the "facility" and "button" and then generate every .sub file from 0 to 65535 with the values entered.

Because these systems are block coded most of the time there could also be an option to choose a starting value and an ending value and an option to skip every 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10 codes, etc. Since you can purchase a box of 10 sequential remotes maybe you could save time and skip x amount of codes that could be in the same 10-pack.

Example: If set to skip every 10 values, it would generate 00000.sub, 00010.sub, 00020.sub, 00030.sub, etc... 

What do you think?  Is this request possible?

**Comment by ZIZI51** [comment]
> Hi... I'm interested in brute forcing MegaCode.  I wanted to request a feature that would make it easier since the BruteForce app doesn't have the option that I know of.
> 
> I would need to generate a set of files instead of just one .sub file.  Almost always the "facility" and the "button" is ignored by the receiver and what matters is just the code.  There could be two options to choose a value for the "facility" and "button" and then generate every .sub file from 0 to 65535 with the values entered.
> 
> Because these systems are block coded most of the time there could also be an option to choose a starting value and an ending value and an option to skip every 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10 codes, etc. Since you can purchase a box of 10 sequential remotes maybe you could save time and skip x amount of codes that could be in the same 10-pack.
> 
> Example: If set to skip every 10 values, it would generate 00000.sub, 00010.sub, 00020.sub, 00030.sub, etc... 
> 
> What do you think?  Is this request possible?
